README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/catalyst/moodle-tool_broadcast.svg?branch=master)](https://travis-ci.org/catalyst/moodle-tool_broadcast)
+![master](https://github.com/catalyst/moodle-tool_broadcast/actions/workflows/moodle-ci.yml/badge.svg)
```

Comment:
you probably need something like ![Build Status](https://github.com/catalyst/moodle-tool_broadcast/actions/workflows/ci.yml/badge.svg?branch=master)

Reply:
The `ci.yml` file is called `moodle-ci.yml`. I just followed https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name. I will go with your suggestion but amend the file name.

Reply:
It ends up being the same actually afaict.

Reply:
On https://github.com/catalyst/moodle-tool_broadcast/tree/removetravis anyways.